04-onion-routing.md

```diff
@@ -290,13 +295,11 @@ The reader:
   - If `encrypted_recipient_data` is present:
     - If `blinding_point` is set in the incoming `update_add_htlc`:
       - MUST return an error if `current_blinding_point` is present.
-      - MUST use that `blinding_point` as the blinding point for decryption.
     - Otherwise:
       - MUST return an error if `current_blinding_point` is not present.
-      - MUST use that `current_blinding_point` as the blinding point for decryption.
+      - MUST use that `current_blinding_point` as `E_i` to derive the following blinding point.
```

Comment:
I don't understand what you mean by "following blinding" point here.

Reply:
This is wrong, it's used both to derive the  ss and to derive the next blinding point. Will fix...